scanner/src/main/java/nuvoex/com/scanner/BarCodeActivity.java

```diff
@@ -57,16 +58,23 @@
     private int mBarCodeCount;
 
     ArrayList<String> mPrefetchList;
+    ArrayList<String> mScannedList;
     private String mItemIndicator;
     private MediaPlayer mediaPlayer;
 
+    private enum ValidationResult {
+        INVALID,
+        VALID,
+        SCANNED;
```

Comment:
No need to use semi colon here.
![screen shot 2016-09-06 at 3 07 27 pm](https://cloud.githubusercontent.com/assets/16488048/18268947/adb07b4a-7443-11e6-924a-70e62f99bcfb.png)